apps/mux/src/index.tsx

```diff
@@ -355,13 +369,23 @@ export class App extends React.Component<AppProps, AppState> {
       ({ policy }: { policy: string }) => policy === 'signed'
     );
 
+    const audioOnly =
+      'max_stored_resolution' in asset && asset.max_stored_resolution === 'Audio only'
+        ? true
+        : false;
+
     await this.props.sdk.field.setValue({
+      version: 2,
```

Comment:
What is the version being used for?

Reply:
Versioning the data in the object.

Reply:
Please keep in mind that the stored data should always be backwards compatible. E.g. you should not drop `audioOnly` in v3 as people relying on that data would run into issues.
Adding the version here is fine, just something to consider for whatever you've planned.